Build system not finding edu.wpi.first.apriltag package

The build system is unable to find the apriltag package located at `edu.wpi.first.apriltag`. 

I'm currently writing a wpilibj example which requires the edu.wpi.first.apriltag package. 
Running `./gradlew wpilibjExamples:build ` from the allwpilib root folder returns returns the following error: 
`~/allwpilib/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/differentialdriveposeestimator/Drivetrain.java:38: error: package edu.wpi.first.apriltag does not exist`

**Desktop (please complete the following information):**
 - WPILib Version: N/A
 - OS: MacOs m1 13.0.1
 - Java version: openjdk version "17.0.4" 2022-07-19
 - C++ version: Apple clang version 14.0.0 (clang-1400.0.29.202)

--- a/wpilibcExamples/build.gradle
+++ b/wpilibcExamples/build.gradle
@@ -60,6 +60,7 @@ model {
                     return
                 }
                 lib project: ':wpilibNewCommands', library: 'wpilibNewCommands', linkage: 'shared'
+                lib project: ':apriltag', library: 'apriltag', linkage: 'shared'
                 lib project: ':wpilibc', library: 'wpilibc', linkage: 'shared'
                 lib project: ':wpimath', library: 'wpimath', linkage: 'shared'
                 project(':ntcore').addNtcoreDependency(binary, 'shared')
@@ -89,6 +90,7 @@ model {
                 binaries.all { binary ->
                     lib project: ':wpilibNewCommands', library: 'wpilibNewCommands', linkage: 'shared'
                     lib project: ':wpilibc', library: 'wpilibc', linkage: 'shared'
+                    lib project: ':apriltag', library: 'apriltag', linkage: 'shared'
                     lib project: ':wpimath', library: 'wpimath', linkage: 'shared'
                     project(':ntcore').addNtcoreDependency(binary, 'shared')
                     lib project: ':cscore', library: 'cscore', linkage: 'shared'
@@ -137,6 +139,7 @@ model {
                 binaries.all { binary ->
                     lib project: ':wpilibNewCommands', library: 'wpilibNewCommands', linkage: 'shared'
                     lib project: ':wpilibc', library: 'wpilibc', linkage: 'shared'
+                    lib project: ':apriltag', library: 'apriltag', linkage: 'shared'
                     lib project: ':wpimath', library: 'wpimath', linkage: 'shared'
                     project(':ntcore').addNtcoreDependency(binary, 'shared')
                     lib project: ':cscore', library: 'cscore', linkage: 'shared'
@@ -202,6 +205,7 @@ model {
         withType(GoogleTestTestSuiteBinarySpec) {
             lib project: ':wpilibNewCommands', library: 'wpilibNewCommands', linkage: 'shared'
             lib project: ':wpilibc', library: 'wpilibc', linkage: 'shared'
+            lib project: ':apriltag', library: 'apriltag', linkage: 'shared'
             lib project: ':wpimath', library: 'wpimath', linkage: 'shared'
             project(':ntcore').addNtcoreDependency(it, 'shared')
             lib project: ':cscore', library: 'cscore', linkage: 'shared'

--- a/wpilibjExamples/build.gradle
+++ b/wpilibjExamples/build.gradle
@@ -27,7 +27,7 @@ if (project.hasProperty('onlylinuxathena') || project.hasProperty('onlylinuxarm3
 
 dependencies {
     implementation project(':wpilibj')
-
+    implementation project(':apriltag')
     implementation project(':wpimath')
     implementation project(':hal')
     implementation project(':wpiutil')
@@ -108,6 +108,7 @@ model {
             }
             binaries.all { binary ->
                 lib project: ':wpilibNewCommands', library: 'wpilibNewCommands', linkage: 'shared'
+                lib project: ':apriltag', library: 'apriltag', linkage: 'shared'
                 lib project: ':wpilibc', library: 'wpilibc', linkage: 'shared'
                 lib project: ':wpimath', library: 'wpimath', linkage: 'shared'
                 lib project: ':wpimath', library: 'wpimathJNI', linkage: 'shared'

